ui/application-window.ui

```diff
@@ -38,6 +38,51 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
     <property name="can_focus">False</property>
     <property name="icon_name">media-playback-stop-symbolic</property>
   </object>
+  <object class="GtkPopover" id="pop_menu">
```

Comment:
The menu shouldn't use a GtkButtonBox. It looks ugly :/

Reply:
What should it use?

Reply:
See Music for example: https://gitlab.gnome.org/GNOME/gnome-music/blob/master/data/ui/AppMenu.ui

Reply:
Thanks. I've changed, it's better?